spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/src/test/java/org/springframework/cloud/kubernetes/configuration/watcher/ActuatorRefreshIT.java

```diff
@@ -104,22 +98,14 @@
 	@Before
 	public void setup() throws Exception {
 		this.client = Config.defaultClient();
+		this.client.setHttpClient(this.client.getHttpClient().newBuilder().readTimeout(Duration.ofSeconds(15)).build());
```

Comment:
why this change?

Reply:
similar reason to https://github.com/spring-cloud/spring-cloud-kubernetes/pull/677#discussion_r534229831. the default read timeout is 5 sec in the okhttp client, in a slower development environment the kind cluster can take longer time to response the test suite..

Reply:
We should probably extract this code to create the client in a common place so it is not repeated in every test.